stride/physics/boundaries/devito.py

```diff
@@ -5,6 +5,43 @@
 from .boundary import Boundary
 
 
+class SpongeBoundaryElastic(Boundary):
+
+    def __init__(self, grid):
+        '''
+        Sponge boundary for elastic codes
+        :param parameter: 0.2763 for homogeneous water, 0.008635 alpha2D
+        '''
+        super().__init__(grid)
+        print('======== Elastic')
+
+        self.damp = None
+
+    def apply(self, field, velocity, parameter):
```

Comment:
use the same signature as the other classes:

def apply(self, field, velocity, direction='forward', **kwargs):

Reply:
Correction made, check line22 is adequate

Reply:
Will use the damping method as suggested below

Reply:
resolved